products/samsungmobile.md

```diff
@@ -1,5 +1,8 @@
 ---
 permalink: /samsung-mobile
+alternate_urls:
+-   /samsung
```

Comment:
-1 for me 👎 
Samsung is a company , this looks like adding microsoft as url to windows